### ISSUE TITLE
`Error performance is not defined ReferenceError: performance is not defined`

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2439]
Update min node version to 17.4.0+

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.30.2](https://github.com/open-wa/wa-automate-nodejs/compare/4.30.1...4.30.2) (2022-01-26)

- Release 4.30.2 [`185472f`](https://github.com/open-wa/wa-automate-nodejs/commit/185472ff116b12dc65993a231ee697038acac6e3)
- ♻️ refactor all instances of `performance` to fallback to `Date` if undefined #2439 [`fb3d662`](https://github.com/open-wa/wa-automate-nodejs/commit/fb3d662bfafab34faa0d89814a3f6a4944a93570)
- ⬇️ revert back to `"node": "&gt;=12.18.3"` #2439 [`042bfb4`](https://github.com/open-wa/wa-automate-nodejs/commit/042bfb491e85df668ce56119bb3b921d663ea598)
- 📌 bump min node version to 17.4.0+ #2439 [`96d252a`](https://github.com/open-wa/wa-automate-nodejs/commit/96d252a28d68e0f35dcae72411b7a64646f5d400)
- 🥚 patches update [`bb204cc`](https://github.com/open-wa/wa-automate-nodejs/commit/bb204ccee3ef4d5881c3d03bafb75c6bd29ea411)
- 🥚 patches update [`ef76e00`](https://github.com/open-wa/wa-automate-nodejs/commit/ef76e00e25ea01519b79cd6d8fae58dfa129ffd2)
- updated types-only package [`8e9e93c`](https://github.com/open-wa/wa-automate-nodejs/commit/8e9e93c12b6b919069a63a2d9b196e534c38541f)

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.30.5](https://github.com/open-wa/wa-automate-nodejs/compare/4.30.4...4.30.5) (2022-01-27)

- 🚑 attempt fix #2439 [`#2439`](https://github.com/open-wa/wa-automate-nodejs/issues/2439)
- Release 4.30.5 [`a997575`](https://github.com/open-wa/wa-automate-nodejs/commit/a9975759e292e284b04d6678e000f99ec2d9d904)
- updated types-only package [`7c36a7e`](https://github.com/open-wa/wa-automate-nodejs/commit/7c36a7e2b7c0a470daedecd42dc7ec0649be0138)
